templates/default/redis.conf.erb

```diff
@@ -250,6 +250,69 @@ dir <%=@datadir%>
 #    but to INFO and SLAVEOF.
 #
 slave-serve-stale-data <%=@slaveservestaledata%>
+<% if @version[:major].to_i == 2 && @version[:minor].to_i >= 6 %>
```

Comment:
This conditional logic doesn't seem match the comment
`# Since Redis 2.6 by default slaves are read-only`